applications/FluidDynamicsHydraulicsApplication/custom_utilities/hydraulic_fluid_auxiliary_utilities.h

```diff
@@ -0,0 +1,151 @@
+     * @param rDistanceVariable Reference to the variable containing the distance
+     * @param IsHistorical True if the distance is in the historical database, false otherwise
+     * @return
+     */
+    static double CalculateWettedPetimeter(
+        ModelPart &rModelPart,
+        const Flags &rSkinFlag,
+        const Variable<double> &rDistanceVariable,
+        const bool IsHistorical);
+    ///@}
+    ///@{
+    /**
+     * @brief This functions calculates the wetted area of a given condition in order to apply the
+     * corresponding inlet boundary condition.
+     * @param rModelPart Inlet Model Part
+     * @param rSkinFlag Flag that marks the conditions to be included in the calculation
+     * @param rDistanceVariable Reference to the variable containing the distance
+     * @param IsHistorical True if the distance is in the historical database, false otherwise
+     * @return
+     */
+    static double CalculateWettedArea(
+        ModelPart &rModelPart,
+        const Flags &rSkinFlag,
+        const Variable<double> &rDistanceVariable,
+        const bool IsHistorical);
+    ///@}
+    ///@{
+    /**
+     * @brief Checks if an element is split
+     * From the given vector containing the distance in each node, this method checks if the element is split
+     * @param rConditionDistancesVector Vector containing the distance values at each node
+     * @return true The element is split
+     * @return false The element is not split
+     */
+    static inline bool IsSplit(const Vector &rConditionDistancesVector);
+
+    /**
+     * @brief Checks if an element is negative
+     * From the given vector containing the distance in each node, this method checks if the element is negative
+     * @param rConditionDistancesVector Vector containing the distance values at each node
+     * @return true The element is negative
+     * @return false The element is not negative
+     */
+    static inline bool IsNegative(const Vector &rConditionDistancesVector);
+
+    /**
+     * @brief Calculates initial water depth guess by taking the average between the maximum and minimum coordinates.
+     *@param rModelPart Inlet Model Part
+     * @return Initial water depth guess
+     */
+    static double InitialWaterDepth(ModelPart &rModelPart);
+
+    /**
+     * @brief
+     * Assign the inlet velocity to all nodes that are wet in the input model part. For dry nodes(air)is assumed that the inlet velocity is null.
+     * @param  rModelPart Inlet Model Part
+     * @param  InletVelocity the velocity value to be assigned to wet nodes.
+     * @param  rDistancesVariable Variable name of the inlet distance.
+     */
+    static void SetInletVelocity(ModelPart &rModelPart, double InletVelocity, const Variable<double> &rDistanceVariable);
+
+    /**
+     * @brief Free the inlet velocity in the nodes belonging to inlet model part.
+     * @param  rModelPart Inlet Model Part
+     */
+    static void FreeInletVelocity(ModelPart& rModelPart);
```

Comment:
```suggestion
    static void FreeInlet(ModelPart& rModelPart);
```
(see the comment above)

Reply:
The name is modified